packages/safe-core-sdk/src/managers/contractManager.ts

```diff
@@ -82,6 +96,52 @@ class ContractManager {
   get multiSendContract(): MultiSendContract {
     return this.#multiSendContract
   }
+
+  private async getSafeContract({
+    ethAdapter,
+    safeVersion,
+    chainId,
+    safeAddress,
+    isL1SafeMasterCopy,
+    customContracts
+  }: GetSafeContractInstanceProps): Promise<GnosisSafeContract> {
+    const safeSingletonDeployment = getSafeContractDeployment(
+      safeVersion,
+      chainId,
+      isL1SafeMasterCopy
+    )
+    const temporarySafeContract = ethAdapter.getSafeContract({
+      safeVersion: SAFE_LAST_VERSION,
+      chainId,
+      singletonDeployment: safeSingletonDeployment,
+      customContractAddress: safeAddress,
+      customContractAbi: customContracts?.safeMasterCopyAbi
+    })
+    if ((await ethAdapter.getContractCode(temporarySafeContract.getAddress())) === '0x') {
+      throw new Error('Safe Proxy contract is not deployed in the current network')
+    }
+    return temporarySafeContract
+  }
+
+  private async getMultiSendContract({
+    ethAdapter,
+    safeVersion,
+    chainId,
+    customContracts
+  }: GetMultiSendContractInstanceProps): Promise<MultiSendContract> {
+    const multiSendDeployment = getMultiSendContractDeployment(safeVersion, chainId)
+    const multiSendContract = await ethAdapter.getMultiSendContract({
+      safeVersion,
+      chainId,
+      singletonDeployment: multiSendDeployment,
+      customContractAddress: customContracts?.multiSendAddress,
+      customContractAbi: customContracts?.multiSendAbi
+    })
+    if ((await ethAdapter.getContractCode(multiSendContract.getAddress())) === '0x') {
```

Comment:
You're checking the address against `0x` twice here. Is it done elsewhere as well? Might be worth creating a helper function?

Reply:
It's also present in `packages/safe-core-sdk/src/managers/ownerManager.ts`